src/components/Shared/Filters/FilterListItemMultiselect.tsx

```diff
@@ -122,7 +127,59 @@ export const FilterListItemMultiselect: React.FC<Props> = ({
             />
           </ListItem>
         ))
-      )}
+      ) : filter.filterKey === 'status' ||
+        filter.filterKey === 'contact_status' ? (
+        <ListItem>
+          <Autocomplete
+            multiple
+            autoHighlight
+            autoSelect
+            value={selected || []}
+            onChange={(_, value) => toggleValue(value)}
+            options={
+              filter.options
+                ?.slice()
+                .sort(
+                  (a, b) =>
+                    statusArray
```

Comment:
I think this could be clearer if we extracted `statusArray?.find((status) => status.id === option.value)` into a helper function that we called once for `a` and once for `b`.